Source/WebCore/page/Quirks.cpp

```diff
@@ -1716,6 +1716,19 @@ bool Quirks::shouldDisableNavigatorStandaloneQuirk() const
     return false;
 }
 
+// booking.com https://webkit.org/b/269875
+// FIXME: booking.com https://webkit.org/b/269876 when outreach has been successful.
+bool Quirks::shouldSendLongerAcceptHeaderQuirk(StringView host, LocalFrame* frame)
+{
+    if (frame && !frame->settings().needsSiteSpecificQuirks())
+        return false;
+
+    if (equalLettersIgnoringASCIICase(host, "www.booking.com"_s))
```

Comment:
If I search for a flight, I'll end up with a URL like "https://flights.booking.com", which is no longer going to match. Isn't this an issue? Is there a reason we only want to do this for "www.booking.com" and not "*.booking.com"?

Reply:
surprisingly flights.booking.com doesn't send JPEG2000 images.

Reply:
also flights.booking.com seems to redirect to www.booking.com at a point

Reply:
booking.com seems to have a lot of subdomains:
https://www.vedbex.com/subdomain-finder/booking.com

Reply:
Modified the PR! Thanks.